File: Edinburgh College/JS/5. Functions/activity3.js
function primeNumber() {
    if (userNumber < 2) {
        console.log("Cannot be less than 2");
    } else {
        for (i = 2; i < userNumber; i++) {
            // If no remainder after division, number is not prime number
            if (userNumber % i == 0) {
                console.log(`${userNumber} is not a prime number`);
                return;
            } else {
                // Number is a prime number
                console.log(`${userNumber} is a prime number`);
                return;
            }
        }
        console.log(`${userNumber} is a prime number`);
    } 
}

// Get user number
const userNumber = Number(prompt("Enter a number: "));

// Run function with user input as parameter
primeNumber(userNumber)

File: Edinburgh College/Python/5. Python Functions/activity3.py
# Function to check prime numbers
def prime_number(number):
    # Check if valid number
    if number < 1:
        return "Invalid number"
    elif number == 1:
        return "Not prime number as only one factor"
    # Loop through numbers
    for integer in range(2, number):
        # Check if remainder
        if number % integer == 0:
            return "Not prime number"
        else:
            return "Prime number"


# Get user input
user_number = int(input("Enter number: "))
# Call function and print result
print(prime_number(user_number))
